test/e2e/tests/tokens/import-tokens.spec.ts

```diff
@@ -83,4 +99,132 @@ describe('Import flow', function () {
       },
     );
   });
+
+  it('allows importing multiple tokens from search across chains', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withNetworkControllerOnMainnet()
+          .withNetworkControllerOnPolygon()
+          .withTokensController({
+            tokenList: [],
+            tokensChainsCache: {
+              '0x1': {
+                data: {
+                  '0x0a0e3bfd5a8ce610e735d4469bc1b3b130402267': {
+                    name: 'Entropy',
+                    aggregators: ['Lifi', 'Coinmarketcap', 'Rango'],
+                    address: '0x0a0e3bfd5a8ce610e735d4469bc1b3b130402267',
+                    decimals: 18,
+                    iconUrl:
+                      'https://static.cx.metamask.io/api/v1/tokenIcons/1/0x0a0e3bfd5a8ce610e735d4469bc1b3b130402267.png',
+                    occurrences: 3,
+                    symbol: 'ERP',
+                  },
+                },
+              },
+              '0x89': {
+                data: {
+                  '0xc2132D05D31c914a87C6611C10748AEb04B58e8F': {
+                    name: 'USDT',
+                    aggregators: ['Lifi', 'Coinmarketcap', 'Rango'],
+                    address: '0xc2132D05D31c914a87C6611C10748AEb04B58e8F',
+                    decimals: 6,
+                    iconUrl:
+                      'https://static.cx.metamask.io/api/v1/tokenIcons/137/0xc2132D05D31c914a87C6611C10748AEb04B58e8F.png',
+                    occurrences: 3,
+                    symbol: 'USDT',
+                  },
+                },
+              },
+            },
+          })
+          .build(),
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockPriceFetch,
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+
+        const homePage = new HomePage(driver);
+        await homePage.check_pageIsLoaded();
+
+        const assetListPage = new AssetListPage(driver);
+
+        await assetListPage.importMultipleTokensBySearch(['ERP', 'USDT']);
+
+        const tokenList = new AssetListPage(driver);
+        await tokenList.check_tokenItemNumber(5); // Polygon, Eth, linea, USDT, ERP
+
+        await tokenList.check_tokenExistsInList('Ethereum');
+        await tokenList.check_tokenExistsInList('ERP');
+        await tokenList.check_tokenExistsInList('USDT');
+        await tokenList.check_tokenExistsInList('POL');
```

Comment:
Should we add more checks in this E2E to ensure they are added to the correct network by checking the asset list UI?